src/requirements/data/majors/ece.ts

```diff
@@ -74,27 +65,8 @@ const eceRequirements: readonly CollegeOrMajorRequirement[] = [
     perSlotMinCount: [3],
   },
   {
-    name: 'Probability and Statistics',
```

Comment:
Looking at the handbooks, it looks like this was a requirement until 2021, when it was fully removed. @einc how can we handle this requirement for pre-2021 users without just deleting it?